### PR TITLE
options renaming, upgrader, and roxygen2

### PR DESCRIPTION
**Pull request by vandenman**
Fixes https://github.com/jasp-stats/INTERNAL-jasp/issues/1986

### CODE REVIEW

File: inst/qml/JAGS.qml [hunk 1]
```diff
@@ -78,15 +78,15 @@ Form
...
+			CheckBox { name: "histogramPlot";		label: qsTr("Histogram")														}
+			CheckBox { name: "tracePlot";			label: qsTr("Trace");															}
 		}
 		Group
 		{
-			CheckBox { label: qsTr("Autocorrelation");    name: "plotAutoCor"; id: autoCorrelation
+			CheckBox { label: qsTr("Autocorrelation");	name: "autoCorPlot"; id: autoCorrelation
 				IntegerField
 				{
 					name: "noLags"
```
Comment (Kucharssim) [inline inst/qml/JAGS.qml:92]:
```suggestion
					name: "autoCorPlotLags"
```

**Review by Kucharssim — CHANGES_REQUESTED** [review]
Looks good, just one suggestion!

**Review by Kucharssim — APPROVED** [review]
Looks good! The options are upgraded correctly, but running a test using `--unitTest` errors out because of little differences in the output (presumably due to sampling variation) . It is odd because I tried to use the `setSeed` option which works fine within one specific version, but different versions of the modules lead to slightly different results... If that's known and not an issue (aka we cannot fix this behavior), we can merge this.

**Comment by vandenman** [comment]
> It is odd because I tried to use the setSeed option which works fine within one specific version, but different versions of the modules lead to slightly different results... If that's known and not an issue (aka we cannot fix this behavior), we can merge this.

If you made the original file with the released 0.16.3 (or used one from the data library), then the shipped Jags version is probably different. Also, the seed is a bit of a lie since Jags isn't reproducible at all across OSes, so any data library file is probably not reproducible ever.

**Pull request merged**